src/components/table/table.tsx

```diff
@@ -98,6 +112,11 @@ export class Table {
         this.tabulator = new TabulatorTable(table, options);
     }
 
+    @Watch('activeRow')
+    public activatedRowChanged() {
+        this.tabulator.redraw(true);
```

Comment:
If you scroll down in a table and click on a row, the table will scroll to the top. I *guess* that's its because of this line :disappointed: